tests/e2e-integration/e2e/sections/home/Home.spec.ts

```diff
@@ -6,39 +6,79 @@ describe('Home Page', () => {
   const title = faker.lorem.sentence()
   before(() => {
     TestsUtils.setup()
+    TestsUtils.login()
+  })
+
+  beforeEach(() => {
+    TestsUtils.login()
   })
 
-  beforeEach(() => {})
   it('successfully loads', () => {
     cy.visit('/spa')
     cy.findAllByText(/Root/i).should('exist')
   })
 
-  it('goes to dataset page from list', () => {
-    void DatasetHelper.destroyAll()
-    void DatasetHelper.createWithTitle(title)
-    TestsUtils.login()
-    cy.findByText(/Dataverse Admin/i).should('exist')
-    cy.findByText(title).should('be.visible')
-    cy.findByText(title).click({ force: true })
-    cy.url().should('include', 'persistentId')
-    cy.findAllByText(title).should('be.visible')
+  it('navigates to a dataset from the list when clicking the title', () => {
+    cy.wrap(
+      DatasetHelper.destroyAll().then(() => DatasetHelper.createWithTitle(title)),
+      { timeout: 6000 }
```

Comment:
```suggestion
      { timeout: 10000 }
```
The test was failing because the timeout expired before destroyAll() completed.  We have another ticket for adding destroyAll() to more tests, to avoid having too many to delete at once, but as quick fix, increasing the timeout to 1000 worked for me locally